src/plugins/data/config.ts

```diff
@@ -49,6 +49,9 @@ export const configSchema = schema.object({
       }),
     }),
   }),
+  searchUsageTelemetry: schema.object({
```

Comment:
do you think it's worth rolling this config into `search` config on 42 so it would be callable as:

```
search.usageTelemetry.enabled
```

Reply:
that are add to the telemetry config and import the telemetry config.

Reply:
I moved the usageTelemetry under search, please take a look and let me know if it is correct, Thanks